test/e2e/data/infrastructure-vsphere/main/ownerreferences/failure-domains.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+kind: VSphereFailureDomain
+metadata:
+  name: "fd1"
+  namespace: "${NAMESPACE}"
+spec:
+  region:
+    name: '${VSPHERE_DATACENTER}'
+    type: Datacenter
+    tagCategory: k8s-region
+    autoConfigure: true
+  zone:
+    name: "cluster0"
```

Comment:
I would add VSPHERE_CLUSTER as a variable and set the default in the e2e configs so it works for us